src/build/tenscript/build_phase.rs

```diff
@@ -217,49 +215,49 @@ impl BuildPhase {
             });
         } else if let Some(node) = node {
             let (node_buds, node_marks) =
-                self.execute_node(fabric, IdentifiedFace { face_id }, &node, vec![]);
+                Self::execute_node(fabric, IdentifiedFace { face_id }, &node, vec![]);
             buds.extend(node_buds);
             marks.extend(node_marks);
         };
         (buds, marks)
     }
 
-    fn execute_node(&self, fabric: &mut Fabric, launch: Launch, node: &BuildNode, faces: Vec<UniqueId>) -> (Vec<Bud>, Vec<FaceMark>) {
+    fn execute_node(fabric: &mut Fabric, launch: Launch, node: &BuildNode, faces: Vec<UniqueId>) -> (Vec<Bud>, Vec<FaceMark>) {
```

Comment:
Cool

Reply:
This was whispered in my ear by the tooling.  Brilliant!